packages/components/pages/f-account-info/src/components/AccountInfo.vue

```diff
@@ -300,6 +316,18 @@ export default {
         */
         setSubmittingState (isFormSubmitting) {
             this.isFormSubmitting = isFormSubmitting;
+        },
+
+        /**
+         * Send through the field & value that has been edited.
+         *
+         * @TODO store in vuex store once that is wired up.
+         *
+         * @param field
+         * @param value
+         */
+        editConsumerDetails (field, value) {
```

Comment:
Will it make sense to listen to the `form` `change` event rather than each field?

Reply:
Valid point. Will give this a try 👍

Reply:
Have fixed this now.

Reply:
Have decided to revert this one as i think it's a little clearer when the method is called on each field, with it at the form level it's a little more obscure as to the field name and target value we're passing. For readability will keep as is if that is ok.